mapeditor/mainwindow.ui

```diff
@@ -1025,36 +1025,36 @@
     <string>Open</string>
    </property>
    <property name="shortcut">
-    <string notr="true">Ctrl+O</string>
+    <string>Ctrl+O</string>
    </property>
   </action>
   <action name="actionOpenRecent">
-    <property name="text">
-     <string>Open Recent</string>
-    </property>
+   <property name="text">
+    <string>Open Recent</string>
+   </property>
   </action>
   <action name="actionOpenRecentMore">
-    <property name="text">
-     <string>More...</string>
-    </property>
-    <property name="shortcut">
-     <string notr="true">Ctrl+R</string>
-    </property>
+   <property name="text">
+    <string>More...</string>
+   </property>
+   <property name="shortcut">
+    <string notr="true">Ctrl+R</string>
```

Comment:
in other places you've enabled translation

Reply:
Yes, thats the way to enable it's visibility in menu.

![image](https://github.com/user-attachments/assets/bb1cd34a-dbc4-4508-9175-0804bef2eaaa)

Reply:
sounds weird...